integration/feature/full-run-logs/src/FullRunLogsTests.scala

```diff
@@ -30,6 +30,10 @@ object FullRunLogsTests extends UtestIntegrationTestSuite {
         normalized,
         List(
           "[info] compiling <digits> Scala sources to .../out/mill-build/compile.dest/classes ...",
+          "[warn] package scala contains object and package with same name: caps.",
```

Comment:
About these warnings, I'm not sure what triggers them. I had a quick look at the meta-build compile path in those tests, but couldn't find a duplicated scala3-library JAR.

Reply:
@alexarchambault got it, could you investigste to see how to make them go away? Presumably if they're showing up in tests they may show up in user terminals as well

Reply:
The was JARs (like the one of scala-collection-compat) that ended up being embedded as is in some coursier JARs, because of issues in code handling shading in the coursier build. Hence some duplicated namespaces, and the warning. https://github.com/coursier/coursier/pull/3487 should fix that.